agoradatatools/etl/load.py

```diff
@@ -151,12 +156,15 @@ def df_to_csv(df: pd.core.frame.DataFrame, filename: str):
 
 def dict_to_json(df: dict, filename: str):
     try:
-        df_as_dict = [{d: remove_non_values(v) if isinstance(v, dict) else v for d,v in df.items()}]
-
-        temp_json = open("./staging/" + filename, 'w+')
-        json.dump(df_as_dict, temp_json,
-                  cls=NumpyEncoder,
-                  indent=2)
+        df_as_dict = [
```

Comment:
Can we add a TODO comment to explore the `df.to_dict` function?

Reply:
added